src/common/index.ts

```diff
@@ -105,3 +105,10 @@ const confirmOptions: ConfirmOptions = {
 export function useProvider(connection: Connection, wallet: any): Provider {
   return useMemo(() => new Provider(connection, wallet, confirmOptions), [connection, wallet, confirmOptions])
 }
+
+export function checkNull(value: any): void {
+  if (!value) {
```

Comment:
Falsey checks are only safe on { } and [ ] objects, but not numbers, strings, or the any type
```suggestion
  if (value == undefined) {
```